app/src/main/java/com/infomaniak/mail/ui/main/menuDrawer/MenuDrawerFragment.kt

```diff
@@ -89,113 +97,25 @@ class MenuDrawerFragment : MenuFoldersFragment(), MailboxListFragment {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        displayVersion()
+        bindAlertToViewLifecycle(inputDialog)
+
         setupListeners()
         setupCreateFolderDialog()
+        setupRecyclerView()
 
+        observeAllTheThingsSheSaid()
```

Comment:
Even though it's very precise, concise and easy to understand, I'm not sure the world is ready for this naming

Reply:
🤣 
Sorry, totally forgot to use a real name.
I was listening to some russian pop at the time.